app/src/ble.c

```diff
@@ -222,6 +222,23 @@ int zmk_ble_clear_bonds() {
     return 0;
 };
 
+int zmk_ble_clear_all_bonds() {
+    LOG_DBG("zmk_ble_clear_all_bonds()");
+
+    // Unpair all profiles
+    for (uint8_t i = 0; i < ZMK_BLE_PROFILE_COUNT; i++) {
+        if (bt_addr_le_cmp(&profiles[i].peer, BT_ADDR_LE_ANY)) {
```

Comment:
This duplicates code from `zmk_ble_clear_bonds()`.

@petejohanson what do you think about this?
1. Rename `zmk_ble_clear_bonds()` to `zmk_ble_clear_active_profile_bonds()`
2. Pull out this `if` block into a static function for clearing bonds for a specific profile given its index.
3. Change `zmk_ble_clear_bonds()` to call that function on `active_profile`, and change this function to call it on every profile.

Reply:
Agreed. We should try to avoid duplicating this, especially if more comes up later that needs to happen when a  given profile is cleared.